admin-core/src/main/kotlin/com/epam/drill/admin/DrillApplication.kt

```diff
@@ -113,6 +121,21 @@ fun Application.moduleWithOAuth2() {
     routing {
         drillAdminRoutes()
         configureOAuthRoutes()
+        route("/api") {
+            authenticate("jwt") {
+                userInfoRoute()
```

Comment:
I didn't add sign-in, sign-up, update-password, reset-password routes because they contradict external authentication way